formatTest/typeCheckedTests/expected_output/attributes.rei

```diff
@@ -11,29 +11,32 @@ let test: int;
 /**
  * #990: don't strip attributes in interface files
  */
-let x: int [@@bs.val];
+[@@bs.val] let x: int;
```

Comment:
Yeah, perhaps unless there is no payload so that we could do cool ppx transforms like:

```
@async let foo = (x, y) => ...;
```